backend/internal/routes/repo_test.go

```
@@ -182,6 +182,47 @@ func TestRepo_UpdateWaypoint_UnknownIsNotFound(t *testing.T) {
+func TestRepo_DeleteWaypoint_TakesItsAttachmentsWithIt(t *testing.T) {
+	db, repo, route := newRepoWithRoute(t)
+	svc := NewService(repo)
+	ctx := context.Background()
+	waypoint := addWaypoints(t, svc, route.ID, "Kandy")[0]
+	require.NoError(t, svc.AttachTasks(ctx, waypoint.ID, []string{seedTask(t, db, route.EventID, "T1")}))
+
+	_, err := svc.DeleteWaypoint(ctx, waypoint.ID)
+	require.NoError(t, err)
+
+	var remaining int
+	require.NoError(t, db.QueryRow(
```

golangci-lint (2.12.2)

[error] (*database/sql.DB).QueryRow must not be called. use (*database/sql.DB).QueryRowContext

(noctx)